### ISSUE TITLE
shadow.service: failed

### ISSUE THREAD

**Issue by LionNatsu** [AOSC-Dev/aosc-os-abbs#502]
pack: user 'locate', 'kdm', 'lightdm', directories do not exist.

/var/lib/{mlocate, kdm, lightdm}

**Comment by MingcongBai** [comment]
A journal would help... Please.

**Comment by LionNatsu** [comment]
@MingcongBai It was almost the same as what I said

**Comment by MingcongBai** [comment]
Oh I remember now. I thought I have told you about these, the way to fix this is to remove those directories, as they were mistakenly packaged into one of the older KDE tarballs - which you happened to be using.

**Comment by LionNatsu** [comment]
@MingcongBai Thanks, I will remove them from my computer in the morning. (Please keep issue be pending)

**Comment by LionNatsu** [comment]
Looks good (for weeks).

**Comment by LionNatsu** [comment]
Reopening due to
```
Nov 04 00:00:00 Lion-Laptop systemd[1]: Starting Verify integrity of password and group files...
Nov 04 00:00:00 Lion-Laptop pwck[16729]: 用户“http”：目录 /srv/http 不存在
Nov 04 00:00:00 Lion-Laptop pwck[16729]: 用户“locate”：目录 /var/lib/mlocate 不存在
Nov 04 00:00:00 Lion-Laptop pwck[16729]: pwck：无改变
Nov 04 00:00:00 Lion-Laptop systemd[1]: shadow.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
```

**Comment by LionNatsu** [comment]
I found this in ***stub***.
```
root@lionbot [ ~ ] # pwck -r  
user 'dbus': directory '/var/run/dbus' does not exist        
user 'http': directory '/srv/http' does not exist            
user 'locate': directory '/var/lib/mlocate' does not exist   
user 'polkitd': directory '/etc/polkit-1' does not exist     
user 'named': directory '/var/named' does not exist          
user 'unbound': directory '/etc/unbound' does not exist      
no matching password file entry in /etc/passwd               
delete line 'postgres:!:16646:0:99999:7:::'? No              
no matching password file entry in /etc/passwd               
delete line 'pulse:!:16754:0:99999:7:::'? No                 
pwck: no changes              
root@lionbot [ ~ ] !
```

**Comment by LionNatsu** [comment]
Has new stub been fixed?

**Comment by MingcongBai** [comment]
I would believe so, but I will mark this issue as `longterm`, as there are certainly other packages with this issue.

**Comment by MingcongBai** [comment]
Still an issue.

**Comment by MingcongBai** [comment]
Addressed with #2882. Closing.
